seeds/_sector/dex/_schema.yml

```diff
@@ -44,3 +44,36 @@ seeds:
         token_sold_address: varbinary
         token_bought_amount_raw: uint256
         token_sold_amount_raw: uint256
+
+  - name: curvefi_celo_base_trades_seed
+    config:
+      tags: dunesql
+      column_types:
+        version: varchar
+        tx_hash: varbinary
+        token_bought_address: varbinary
+        token_sold_address: varbinary
+        token_bought_amount_raw: uint256
+        token_sold_amount_raw: uint256
+
```

Comment:
Hi @tomfutago, would you mind adding a new column `block_number` in your seed files?
In this case the seed files will keep consistent as in #4774 .

Reply:
sure, will do